public/sh/install-wildfly

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+
+REPO="wildfly/wildfly"
+
+if [ -n "${WILDFLY_VERSION}" ]; then
+  echo "🔍 Fetching ${WILDFLY_VERSION} release from GitHub..."
+else
+  echo "🔍 Fetching latest release from GitHub..."
+  WILDFLY_VERSION=$(curl -s https://api.github.com/repos/$REPO/releases/latest \
```

Comment:
Do you know what this will be in the period between a beta and its Final?

I'm too lazy to research, so Ill ask first. ;)

Reply:
I've also updated our [Release Procedure](https://docs.google.com/document/d/1Bjefo56-J_8PtPxTbHCiE-M3TeelHjxKppaJMmu6HSE/edit?tab=t.0#heading=h.bc5yh5p35cgb) to call out when these checkboxes ("pre-release", "latest" must be checked when we publish a release)